pkg/pillar/hypervisor/containerd.go

```diff
@@ -19,20 +19,23 @@ type ctrdContext struct {
 	// XXX add log?
 	domCounter int
 	PCI        map[string]bool
+	ctrdClient *containerd.Client
 }
 
-func initContainerd() (*ctrdContext, error) {
-	if err := containerd.InitContainerdClient(); err != nil {
+func initContainerd(createClient bool) (*ctrdContext, error) {
```

Comment:
Seems like all callers call with true so the createClient argument can be removed.

Reply:
Removed createClient.